content/blog/posts/o3de-interactive-tutorials-prototyping-2022.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Advance Prototyping of the InteractiveTutorials Gem for O3DE"
+description: ""
+date: 2022-08-10
+slug: ""
+author: Madeleine Song
+blog_img: "/images/blog/interactive-tutorials-prototyping/interactive-tutorials.png"
```

Comment:
I would suggest using a different image for this one as its getting messed up on the blog list. 
![dev-series112](https://user-images.githubusercontent.com/80487462/184217288-5b51d2d7-399c-48ef-894b-2a810af55b7d.JPG)